src/fhirConfig.ts

```diff
@@ -82,10 +82,6 @@ export interface Server {
     url: string;
 }
 
-export interface Logging {
```

Comment:
Removing it as we're passing log level in from Lambda environment

Reply:
This will be a breaking change right?

Reply:
Yes! Good catch!